NeoMathEngine/src/CPU/CpuMathEngineDnnLstm.h

```diff
@@ -0,0 +1,110 @@
+/* Copyright © 2017-2020 ABBYY Production LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--------------------------------------------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <NeoMathEngine/NeoMathEngine.h>
+#include <NeoMathEngine/CrtAllocatedObject.h>
+#include <MemoryHandleInternal.h>
+
+namespace NeoML {
+
+struct CMathEngineLstmDesc : public CLstmDesc {
+	CMathEngineLstmDesc( const CFloatHandle& _inputFullyConnectedResult, const CFloatHandle& _reccurentFullyConnectedResult,
+		int _hiddenSize, int _objectCount, int _objectSize, IMathEngine* _mathEngine, int _threadCount ) :
```

Comment:
_[minor, codestyle]_

multiline argument list (if initializer list is present) offsets on _2 indents_.
every element of initializer list must be on a new line

```c++
CMyClass( long argumentList0, long argumentList1, long argumentList2, long argumentList3,
        long argumentList4, long argumentList5, long argumentList6, long argumentList7 ) :
    initializer0( argumentList0 ),
    initializer1( argumentList1 ),
    initializer2( argumentLis2 ),
    ...
```